pkg/eval/str/str.go

```diff
@@ -331,6 +389,37 @@ func split(fm *eval.Frame, opts maxOpt, sep, s string) {
 // ▶ ABC!123
 // ```
 
+//elvdoc:fn to-utf8-bytes
```

Comment:
It looks like you refactored the change into two commits and this portion of this commit was meant to be in the second commit. Oops! :smile: Since I know what a PITA this type of refactoring is, and both commits are small and closely related I would just squash them into a single commit.

Reply:
I am really sorry for this, probably happened when i tried to fix the formatting with a rebase